configure.ac

```diff
@@ -698,6 +700,7 @@ case "${host}" in
 	if test "${LG_SIZEOF_PTR}" = "3"; then
 	  default_retain="1"
 	fi
+	zero_realloc_default_free="1"
```

Comment:
Last suggestion I might make is to check if it's Linux *and* glibc here; e.g. musl does `alloc` by default.

Reply:
Yeah I remember your comment earlier on this. However at this point I'm hesitant to make that change (plan to publish the release tomorrow), since we don't have a glibc / musl check already in place -- 10 lines above we simply declare `glibc="1"` under Linux.

Reply:
Fair enough, seems reasonable